src/pcapi/core/bookings/repository.py

```diff
@@ -279,7 +280,7 @@ def _duplicate_booking_when_quantity_is_two(bookings_recap_query: Query) -> Quer
     return bookings_recap_query.union_all(bookings_recap_query.filter(Booking.quantity == 2))
 
 
-def _build_bookings_recap_query(user_id: int, venue_id: int = None) -> Query:
+def _build_bookings_recap_query(user_id: int, event_date: Optional[date], venue_id: int = None) -> Query:
```

Comment:
```suggestion
def _build_bookings_recap_query(user_id: int, event_date: Optional[date] = None, venue_id: int = None) -> Query:
```

(comme son copain `venue_id`, même si en toute logique ce n'est pas nécessaire vu qu'il est par défaut au-dessus déjà)

Reply:
Yes comme tu dit elle l'est déjà au dessus. Je l'ai mis "au dessus" puisque c'est la fonction qu'on considère publique, celle-ci n'est censé être appelée que par celle du dessus (ou en tous les cas pas par l'extérieur vu son nom) donc j'ai préféré ne pas surcharger. Peut-être qu'on peut l'enlever de `venue_id` ?